governance/coordination-structure.md

```diff
@@ -26,107 +23,136 @@ Toronto Mesh uses voting to determine whether we have reached consensus:
 
 ## Working Groups
 
+### Responsibilities of Working Group Leads
+
+Central Org
+  - Organize meetups and planning sessions
+  - Post event details to the tomesh.net website 
+
+Tools
```

Comment:
There are more! All of these: https://github.com/tomeshnet/documents/tree/master/service_setup

Reply:
Nice, thanks for the link!